arch/arm/boot/dts/overlays/dacberry400.dts

```diff
@@ -0,0 +1,75 @@
+// Definitions for DACberry400
```

Comment:
This file should be called `dacberry400-overlay.dts`, otherwise it won't be compiled.

Reply:
renamed to dacberry400-overlays.dts